package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pdftron/webviewer-react-toolkit",
-  "version": "0.7.3",
+  "version": "0.7.2",
```

Comment:
did not know doing a release will autoincrement version for you. so i reverted back version.